<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnGenerate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>620</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Сгенерировать 
 график платежей</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выдача микрозайма</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>42</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>home.png</normaloff>home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>351</width>
     <height>519</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="14" column="1">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Страховка</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Улица</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Отчество(при наличии)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Серия паспорта</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_9">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Сумма кредита</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="7" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="dateEdit_2"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Номер паспорта</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Имя</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Кем выдан</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Процентная ставка</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Фамилия</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Дата выдачи</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Дом</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Город</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_8"/>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEdit_9"/>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEdit_10"/>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Количество месяцев</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="lineEdit_11"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnGive">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>620</y>
     <width>164</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Выдать микрозайм</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
